content/articles/switch-vs-router-vs-hub/index.md

```diff
@@ -0,0 +1,117 @@
+In networking the terms "Switches", "Hubs", and "Routers" are sometimes used interchangeably and this is wrong. Despite them being similar, there are differences in how they handle data. These three components may be integrated into a single device making it hard for a student to distinguish between them.
+
+In this article, we are going to discuss each device and their functionalities in a network.
+
+### Switch
+A switch is a multicast networking device that works under the Datalink layer of the OSI model and connects a bunch of computers or devices in a network.
+
+### Advantages of using a Switch
+- It is secure since it delivers data to the specified node.
+- Lowers the chances of frame collisions domains.
+- It increases the bandwidth in a network.
+- It increases the number of ports needed to connect the nodes available in a network.
+- It operates under full-duplex.
+
+### Disadvantages of using Switches
+- They are more expensive compared to hubs and other devices used in a network.
+-  To deal with multicast parcels, proper planning is required.
+- It can raise problems in broadcast traffic.
+
+### Hub
+Hub is a simple and cheap networking device that works under the physical layer of the OSI model and connects a bunch of computers in a Local Area Network(LAN). (It does lots of wastage of bandwidth). A hub is considered less intelligent because it does not filter data and does not know where the data is to be sent.
+
+All information sent to a hub is automatically sent to all ports of the devices connected to it.
+ 
+### Advantages of using hubs
+ - Ability to connect to the network using different physical media.
+ - They can be used to increase the network distance.
+ - Hubs are relatively cheap compared to switches and other devices in the network.
+
+### Disadvantages of using hubs
+- It increases the chances of collision domains between packets when being transferred from one device to another.
+- Hubs operate under half-duplex that's only one occasion of data transmission at a time.
+- Hubs share data to all the devices in a network thus insecure.
+- Hubs does lots of wastage of bandwidth when transmitting data.
+
+### Switch Vs Hub
+Hub is a broadcast device that sends data from one node to all but Switch is a multicast device that can send data to a particular node you want.
+

```

Comment:
I believe there are more comparisons between a switch and a hub. Please add them.